Example/Emission/AppDelegate.m

```diff
@@ -103,7 +103,7 @@ - (void)setupEmissionWithUserID:(NSString *)userID accessToken:(NSString *)acces
                           useStagingEnvironment:useStaging
                                       sentryDSN:nil];
 #else
-#ifdef DEPLOY
+#if DEPLOY
```

Comment:
I might switch the app entirely to just debug and release, I don't think a third schema is useful as release will not ever be used